node/service/src/rpc/tests/mod.rs

```diff
@@ -1,55 +0,0 @@
-mod rpc_mock;
-
-use rpc_mock::*;
```

Comment:
I had to remove this since the setup fundamentally didn't work with the Pool and added the removed test into e2e side.